.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Remove this file and add a .gitignore entry to ignore the .vscode directory, please, as it contains settings for your username and OS, so not valuable for the greater community.